data/json/recipes/armor/storage.json

```diff
@@ -223,6 +223,24 @@
       [ [ "string_6", 3 ], [ "filament", 10, "LIST" ] ]
     ]
   },
+  {
+    "result": "bow_sling",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ "archery", 1 ],
```

Comment:
I'm not certain why, but I feel like this should be Archery 2, 1 just seems too basic/insufficient to being able to figure out a decent way to strap and use bows.

Reply:
Considering pretty much all other holsters and sheathes are level 1 in their respective skill, I think it's fine as is. Honestly you really only need to know the general shape of a bow to make one.